db/migrate/20161028104813_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :firstname
+      t.string :lastname
+      t.string :email
+      t.string :password_digest
+      t.string :iss
```

Comment:
@andela-megwali This property has never been descriptive for once. I keep asking what it means